### PR TITLE
Make colour picker highlight more visible

### PR DESCRIPTION
**Pull request by googletorp**

### CODE REVIEW

File: web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss [hunk 1]
```diff
@@ -79,8 +79,7 @@ $color_schemes: (
 
       label {
         display: block;
-        top: -40px;
-        position: relative;
+        margin: 16px 0 0;
```
Comment (naxoc) [inline web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss:82]: Should be `$spacing * 8`

File: web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss [hunk 2]
```diff
@@ -92,14 +91,17 @@ $color_schemes: (
...
           cursor: pointer;
           -webkit-appearance: none;
           -moz-appearance: none;
+          transform: none;
+          margin: 0;
+          float: none;
 
           &:checked + label {
             text-decoration: underline;
+            font-size: 130%;
```
Comment (naxoc) [inline web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss:104]: Not sure here, but should this not be `rem` instead?

File: web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss (hunk 1 above)
Comment (googletorp) [reply web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss:82]: Backend theme is different :)

File: web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss (hunk 2 above)
Comment (googletorp) [reply web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss:104]: Backend theme...

**Comment by xendk** [comment]
Screenshot? :-)

File: web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss (hunk 1 above)
Comment (naxoc) [reply web/themes/custom/kk_admin/src_assets/scss/components/color_picker.scss:82]: Yikes. Sorry! You're right :)

**Pull request merged**